src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -191,6 +206,11 @@ public void readFrom(StreamInput in) throws IOException {
         unassignedShards = in.readVInt();
         numberOfNodes = in.readVInt();
         numberOfDataNodes = in.readVInt();
+        if (in.getVersion().onOrAfter(Version.V_1_5_0)) {
+            numberOfPendingTasks = in.readVInt();
+        } else {
+            numberOfPendingTasks = 0;
+        }
```

Comment:
can we make this a regular "readInt" call, and use `-1` to denote that we couldn't compute it? otherwise, 0 can be very misleading